server/tests/test_api.py

```diff
@@ -193,6 +208,16 @@ def test_no_data(self):
         rv = self.client.post('/api/0/lectures/1/comments')
         assert rv.status_code == 400
 
+    def test_submission_time(self):
+        rv = self.client.post('/api/0/lectures/1/comments', data=dict(
+            data='hello!'
+        ))
+        assert rv.status_code == 200
+
+        presubmission_time = datetime.utcnow()
```

Comment:
Maybe it should be `postsubmission_time`, as I see it?
Or move it before making request.